Unit Tests fail if PORTER_HOME contains an Uppercase character

## Describe the bug
Hi, I was setting up my development environment to hopefully try contributing via https://porter.sh/contribute/tutorial/ . Most things are fine, but I ran into a curious bug where unit tests fail if the PORTER_HOME contains an uppercase character.
Basically, it looks like some of the test setup is converting the path to a lowercase:
e.g:
```
--- FAIL: TestPorter_Run (0.00s)
    helpers.go:266: open /home/tchaudhry/workspace/porter/pkg/porter: no such file or directory
```
The actual filepath is 'Workspace' instead of 'workspace', so this obviously fails to find the file.

If I clone into a lowercase directory path, all is good.

## To Reproduce
Steps to reproduce the behavior:
1. Clone the repository into a path which contains an uppercase character.
2. Run Unit tests via `make test-unit`

## Expected behavior
Tests should pass :)

## Porter Command and Output
```
15:52:48 tchaudhry@QuasExort porter main ? make test-unit
PORTER_UPDATE_TEST_FILES= GO111MODULE=on go test ./...
?       get.porter.sh/porter/cmd/exec   [no test files]
--- FAIL: TestBuildListMixinsCommand_DefaultFormat (0.00s)
    helpers.go:263: open /home/tchaudhry/workspace/porter/cmd/porter: no such file or directory
--- FAIL: TestBuildListMixinsCommand_AlternateFormat (0.00s)
    helpers.go:263: open /home/tchaudhry/workspace/porter/cmd/porter: no such file or directory
--- FAIL: TestBuildListMixinsCommand_BadFormat (0.00s)
    helpers.go:263: open /home/tchaudhry/workspace/porter/cmd/porter: no such file or directory
--- FAIL: TestBuildMixinInstallCommand (0.00s)
    helpers.go:263: open /home/tchaudhry/workspace/porter/cmd/porter: no such file or directory
--- FAIL: TestBuildMixinInstallCommand_NoMixinName (0.00s)
    helpers.go:263: open /home/tchaudhry/workspace/porter/cmd/porter: no such file or directory
--- FAIL: TestRun_Validate (0.00s)
```
Truncated more failures. 


## Version
Copy the output of `porter version` below
porter v0.38.11-11-g8179df1a (8179df1a)


## Potential Fix?
I did some digging, it looks like this is happening because of the following line:
https://github.com/getporter/porter/blob/8179df1ac77a2f99a2edb59e0215103213fbe7dd/pkg/context/helpers.go#L86
If I remove this line, all works fine. But obviously, it might have been here for a reason, hence the report. 
Could it be want to lowercase the filename (last element), right now it's lowercasing the entire path?
Let me know if this can be put into a PR.
Thanks!

--- a/CONTRIBUTORS.md
+++ b/CONTRIBUTORS.md
@@ -62,3 +62,4 @@ and we will add you. **All** contributors belong here. 💯
 * [Tamir Kamara](https://github.com/tamirkamara)
 * [Chioma Onyekpere](https://github.com/Simpcyclassy)
 * [Hrittik Roy](https://github.com/hrittikhere)
+* [Tanmay Chaudhry](https://github.com/tchaudhry91)

--- a/pkg/context/helpers.go
+++ b/pkg/context/helpers.go
@@ -83,7 +83,6 @@ func (c *TestContext) GetTestDefinitionDirectory() string {
 		if !ok {
 			c.T.Fatal("could not determine calling test directory")
 		}
-		filename = strings.ToLower(filename)
 		if strings.HasSuffix(filename, "_test.go") {
 			return filepath.Dir(filename)
 		}

